tests/std/include/test_header_units_and_modules.hpp

```diff
@@ -241,7 +241,24 @@ void test_flat_map() {
     using namespace std;
     puts("Testing <flat_map>.");
 
-    // FIXME! ADD TEST COVERAGE HERE!
+    [[maybe_unused]] constexpr sorted_unique_t unique_tag         = sorted_unique;
+    [[maybe_unused]] constexpr sorted_equivalent_t equivalent_tag = sorted_equivalent;
+
+    constexpr auto simple_truth = [](const auto&) { return true; };
+
+    flat_map<int, int> fm;
+    fm.emplace(1, 1);
+    fm.emplace(1, 2);
+    assert(fm.size() == 1);

```

Comment:
Suggestion (I am really nitpicking here for such a small test) - in general I recommend "unique values for unique purposes" in tests, so seeing `1` appear as a key, a value, and a count is a little confusing. I'd recommend choosing distinctive values. Occurs below.